src/main/java/com/minecolonies/items/AbstractItemMinecolonies.java

```diff
@@ -10,14 +10,18 @@
  */
 public abstract class AbstractItemMinecolonies extends Item
 {
+    private final String name;
+
     /**
      * Sets the name, creative tab, and registers the item.
      */
-    public AbstractItemMinecolonies()
+    public AbstractItemMinecolonies(String name)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): name [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)